### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by AleksanderKamienski**

### CODE REVIEW

File: evac/worker.py [hunk 1]
```diff
@@ -1042,8 +1044,7 @@ def psql_error(self):
 
 if __name__ == "__main__":
     w = Worker()
-    try:
```
Comment (mmackiewicz97) [inline evac/worker.py:1045]: you can not change this

File: redis_aamks/worker/config.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import os
 redis_host = os.environ['AAMKS_SERVER']
 redis_password = os.environ['AAMKS_REDIS_PASS']
-main_path = "/".join(os.environ['AAMKS_PROJECT'].split('/')[:-3])
+# main_path = "/".join(os.environ['AAMKS_PROJECT'].split('/')[:-3])
```
Comment (mmackiewicz97) [inline redis_aamks/worker/config.py:4]: main_path is used to logger, you should remove #

File: redis_aamks/worker/worker.py [hunk 3]
```diff
@@ -33,14 +33,14 @@ def redis_queue_pop(self, db):
         # the `b` in `brpop` indicates this is a blocking call (waits until an item becomes available)
         _, message = db.brpop(config.redis_queue_name)
         message_json = loads(message)
-        logger.debug(f'pop from head of queue \n{message_json["data"]}')
+        # logger.debug(f'pop from head of queue \n{message_json["data"]}')
```
Comment (mmackiewicz97) [inline redis_aamks/worker/worker.py:36]: maybe logger can stay

**Review by mmackiewicz97 — APPROVED** [review]
looks fine

File: evac/worker.py (hunk 1 above)
Comment (AleksanderKamienski) [reply evac/worker.py]: its easier to debug when there is no try catch in evac/worker.py you can see when exception happens on stderr

File: redis_aamks/worker/config.py (hunk 2 above)
Comment (AleksanderKamienski) [reply redis_aamks/worker/config.py]: Ok, I restore logger. Logger doesnt work on our test environment (83,84 machines) - there is a problem with environment variables. There will be an exception when reading environment variables.

File: redis_aamks/worker/worker.py (hunk 3 above)
Comment (AleksanderKamienski) [reply redis_aamks/worker/worker.py]: Ok, I restore logger. Logger doesnt work on our test environment (83,84 machines) - there is a problem with environment variables. There will be an exception when reading environment variables.

**Pull request merged**